arch/x86/core/intel64/locore.S

```diff
@@ -124,8 +124,10 @@ x86_ap_start:
 
 	lgdt gdt48
 	lidt idt48
-	movw $1, %ax
-	lmsw %ax
```

Comment:
Full disclosure, this may have been an instruction introduced in 1982, but until 2021-05-27 I did not know it existed.